examples/mdl_sdk/dxr/mdl_d3d12/mdl_material_library.cpp

```diff
@@ -72,9 +72,8 @@ Mdl_material_library::Mdl_material_library(
 
 Mdl_material_library::~Mdl_material_library()
 {
+    // Release memory here
     m_target_map.clear();
-    for (auto& entry : m_targets)
-        delete entry.second;
```

Comment:
As described m_targets is no longer for controlling the lifetime of Mdl_material_target*